ext/mbstring/tests/sjis_encoding.phpt

```diff
@@ -0,0 +1,61 @@
+--TEST--
+Exhaustive test of Shift-JIS encoding verification and conversion
+--SKIPIF--
+<?php
+extension_loaded('mbstring') or die('skip mbstring not available');
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+?>
+--FILE--
+<?php
+srand(999); /* Make results consistent */
+include('encoding_tests.inc');
+mb_substitute_character(0x25); // '%'
+
+/* Read in the table of all characters in Shift-JIS */
+readConversionTable(__DIR__ . '/data/SHIFTJIS.txt', $validChars, $fromUnicode);
```

Comment:
Might it make sense to preprocess the data files to drop all the `# Char name` comments? Not sure how much it would help in reducing the size.

Reply:
Ah, I see you did do that for EUC-JP already.